pkg/config/config.go

```diff
@@ -1,32 +1,30 @@
 package config
 
-import (
-	corev1 "k8s.io/api/core/v1"
-)
+import corev1 "k8s.io/api/core/v1"
 
-/* Package config defines the configuration for the NATS user-defined source.
+/* Package config defines the configuration for the Nats user-defined source.
 The configuration includes the URL to connect to NATS cluster, the subject onto which messages are published,
-the queue for queue subscription, the TLS configuration for the NATS client and the authentication information.
+the queue for queue subscription, the TLS configuration for the Nats client and the authentication information.
 */
 
-// Config represents the configuration for the NATS client.
-type Config struct {
-	// URL to connect to NATS cluster, multiple urls could be separated by comma.
+// NatsConfig represents the configuration for the Nats client.
+type NatsConfig struct {
+	// URL to connect to Nats cluster, multiple urls could be separated by comma.
 	URL string `json:"url" protobuf:"bytes,1,opt,name=url"`
 	// Subject holds the name of the subject onto which messages are published.
 	Subject string `json:"subject" protobuf:"bytes,2,opt,name=subject"`
 	// Queue is used for queue subscription.
 	Queue string `json:"queue" protobuf:"bytes,3,opt,name=queue"`
-	// TLS configuration for the NATS client.
+	// TLS configuration for the Nats client.
 	// +optional
-	TLS *TLS `json:"tls" protobuf:"bytes,4,opt,name=tls"`
+	TLS *NatsTLS `json:"tls" protobuf:"bytes,4,opt,name=tls"`
 	// Auth information
 	// +optional
-	Auth *Auth `json:"auth,omitempty" protobuf:"bytes,5,opt,name=auth"`
+	Auth *UDNatsAuth `json:"auth,omitempty" protobuf:"bytes,5,opt,name=auth"`
```

Comment:
will you add the `yaml` tags?

Reply:
I am thinking No for now because we are not planning to use the package. If we decide to use it, we should add the `yaml` tags.